Artroom1/ArtViewController.swift

```diff
@@ -13,83 +13,51 @@ import UIKit
 class ArtViewController: UIViewController {
     
     var chosenArtAttributes: ArtAttributes!
+    var randomIndex = Int()
+    var selection: [Artwork] = []
```

Comment:
As the selection is an internal detail of this view controller (at least currently no other part of the application is interested in this), I suggest to mark it with `private`. This way the compiler can enforce that we can not access this property from outside of this class.

```suggestion
    private var selection: [Artwork] = []
```